docs/getting-started/my-account/README.md

```diff
@@ -8,66 +8,55 @@ keywords: my account, account settings, change language
-| [Email reminders](#email-reminders)                                                           | How to change email reminders sent by OpenProject                           |
-| [Set an Avatar](#set-an-avatar)                                                               | How to set an avatar in OpenProject and change the profile picture          |
-| [Delete account](#delete-account)                                                             | How to delete my own account                                                |
+| [Change password](#change-password)                          | How to change my password                                    |
+| [Two-factor authentication](#two-factor-authentication)      | How to set up a two-factor authentication                    |
+| [Access tokens](#access-tokens)                              |                                                              |
+| [Session management](#session-management)                    | How to manage your OpenProject sessions                      |
+| [Notifications settings](#notifications-settings)            | How to change in-app notifications in OpenProject            |
+| [Email reminders](#email-reminders)                          | How to change email reminders sent by OpenProject            |
+| [Set an Avatar](#set-an-avatar)                              | How to set an avatar in OpenProject and change the profile picture |
+| [Delete account](#delete-account)                            | How to delete my own account                                 |
 
 ## Open My account settings
 
 To open your personal settings in OpenProject, click on your user icon in the top right corner in the header of the application.
 
 Choose **My account**.
 
-![my account profil information](my-account-profil.png)
-
-## Change password
-
-In order to reset your password, navigate to the My account section and choose **Change password** in the menu.
-
-Enter your current password.
-
-Enter your new password and confirm it a second time.
-
-Press the blue **Save** button in order to confirm the password changes.
-
-![my account change password](my-account-change-password.png)
-
-> **Note**: You cannot reset your Google password in OpenProject. If you authenticate with a Google/Gmail account, please go to your Google account administration in order to change your password.
+![my account profil information](openproject_my_account_profile.png)
 
 ## Edit your user information
+To change your email address or your name, navigate to **Profile** on the right side menu of **My account** page.
 
-To change your email address or your name, navigate to the **Profile** page in the menu of your My account settings.
+Here you can update the information and **save** your changes.
+
+![openproject_my_account_profile](openproject_my_account_profile.png)
```

Comment:
@MayaBerd your real data is visible here. Should we fake some data?

Reply:
@marcalcobe I think it's alright, it used to show @birthe's data. @birthe what do you say?

Reply:
changed the screenshot and used a user named "OpenProject User"